Makefile

```diff
@@ -129,6 +129,7 @@ dev: globals ## Set Environment to DEV
 	$(eval export SYSTEM_DNS_ZONE_NAME=${DEPLOY_ENV}.dev.cloudpipeline.digital)
 	$(eval export APPS_DNS_ZONE_NAME=${DEPLOY_ENV}.dev.cloudpipelineapps.digital)
 	$(eval export ALERT_EMAIL_ADDRESS?=govpaas-alerting-dev@digital.cabinet-office.gov.uk)
+	$(eval export NEW_ACCOUNT_EMAIL_ADDRESS=${ALERT_EMAIL_ADDRESS})
```

Comment:
Assuming this is now actually needed, because we're always enabling `user_creation`?

Is the default of `govpaas-alerting-dev@digital.cabinet-office.gov.uk` a bit weird though? Are we just using this as the email because it's already SES validated and we don't have a better one? If so, maybe worth a comment to that effect.

Reply:
This is needed, it turns out (I didn't check before) if we had the multi-cloud email in SES, but we do. So I have added a new commit which changes this